### ISSUE TITLE
Make the approximate PDFs

### ISSUE THREAD

**Issue by aimalz** [aimalz/qp#56]
* After #55 is closed. . .
* read in PDFs via #48
* via #43 produce a catalog of these PDFs in each parametrization (bins, quantiles, samples, mixture model via #51), (possibly using multiprocessing, which would be a new issue)
* export to file via #48

**Comment by aimalz** [comment]
I'm closing this for the same reason as #55.
